pennylane_lightning/core/_serialize.py

```diff
@@ -59,7 +59,12 @@ class QuantumScriptSerializer:
 
     # pylint: disable=import-outside-toplevel, too-many-instance-attributes, c-extension-no-member, too-many-branches, too-many-statements
     def __init__(
-        self, device_name, use_csingle: bool = False, use_mpi: bool = False, split_obs: bool = False
+        self,
+        device_name,
```

Comment:
could we set `device_name` as device info, which could contain the backend info?

Reply:
But `serialize` is used for the other devices. I would suggest not changing the name of `device_name`